avm/ptn/authorization/policy-exemption/tests/e2e/mg.defaults/main.test.bicep

```diff
@@ -0,0 +1,48 @@
+targetScope = 'managementGroup'
+metadata name = 'Policy Exemptions (Management Group scope)'
+metadata description = 'This module deploys a Policy Exemption at a Management Group scope using minimal parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Required. The policy definition ID to assign the policy to.')
+param policyDefinitionID string = '/providers/Microsoft.Authorization/policyDefinitions/06a78e20-9358-41c9-923c-fb736d382a4d'
```

Comment:
This is ideally directly implemented in line 29 instead of introducing a new parameter/variable which is only used in a single location.